frontend/public/components/masthead-toolbar.jsx

```diff
@@ -160,11 +180,17 @@ class MastheadToolbar_ extends React.Component {
     }];
   }
 
-  _renderMenuItems(actions) {
-    return actions.map((action, i) => action.separator
+  _renderMenuItems(actions, appendItem) {
```

Comment:
Better to have the caller do the append instead of making it an argument here

Reply:
Ok, adjusted.